### PR TITLE
Needles exchanged - int fields, backend and frontend changes

### LINKED ISSUE #485: add # exchanged for to SEP visit object
SEP form should include an int field to enter the # people exchanged for (needles in, needles out, # exchanged for).

### LINKED ISSUE #486: add # exchanged for to SEP visit form
SEP form should include an int field to enter the # people exchanged for (needles in, needles out, # exchanged for).

### LINKED ISSUE #487: Needles In, Out, #exchanged for free entry INT fields
SEP data fields should be free entry number only fields

### PR DESCRIPTION
**Pull request by JackRyan1989**
I think this covers everything? But please let me know if I missed something crucial.

I added PrevPointInputs for the int fields on the frontend. Let me know if you would like them styled a bit more.
Backend was pretty straightforward, changed the model, serializer, views, fixtures, and tests for SEP data. Made new migration.
Adding in the NeedlesExchanged field to the sidebar was also pretty straightforward. Validation was pretty much already set up.
No tests written in Jest yet. 

Closes #485 
Closes #486 
Closes #487

### CODE REVIEW

File: core/migrations/0013_sepdata_needles_exchanged.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0012_auto_20201216_0954'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='sepdata',
+            name='needles_exchanged',
```
Comment (MikeyManoguerra) [inline core/migrations/0013_sepdata_needles_exchanged.py:15]: i think the name should be exchanged_for.

" the number of people that are involved with this exchange"

**Review by MikeyManoguerra — CHANGES_REQUESTED** [review]
just changing the name of the variable

File: core/migrations/0013_sepdata_needles_exchanged.py (hunk 1 above)
Comment (JackRyan1989) [reply core/migrations/0013_sepdata_needles_exchanged.py:15]: @MikeyManoguerra Should this change happen on the frontend as well, or are you just concerned about how the data will appear in the database?

Comment (MikeyManoguerra) [reply core/migrations/0013_sepdata_needles_exchanged.py:15]: front end too if you can, try and keep in I sync @JackRyan1989

**Review by MikeyManoguerra — APPROVED** [review]

File: core/migrations/0013_sepdata_needles_exchanged.py (hunk 1 above)
Comment (mariekers) [reply core/migrations/0013_sepdata_needles_exchanged.py]: there should be three SEP fields
1. needles_in: the number of syringes they brought in to exchange
2. needles_out: the number of syringes PPP gave them 
3. exchanged_for: the number of people they are exchanging needles for

**Pull request merged**